doc/cephadm/mgr.rst

```diff
@@ -0,0 +1,36 @@
+.. _mgr-cephadm-mgr:
+
+MGR Service
```

Comment:
Do we need an entry for this new `mgr.rst` in `doc/cephadm/index.rst `to make it reachable?